### PR TITLE
Median flow tracker update

### PR DESCRIPTION
**Pull request by sovrasov**

### CODE REVIEW

**Comment by LeonidBeynenson** [comment]
@vpisarev , could you review it a bit?

File: modules/tracking/src/trackerMedianFlow.cpp [hunk 1]
```diff
@@ -164,28 +164,20 @@ bool TrackerMedianFlowImpl::updateImpl( const Mat& image, Rect2d& boundingBox ){
...
-    default:     r = "User"; break;
-  }
-
-  r += "C";
-  r += (chans+'0');
+struct NaNChecker
+{
+    bool operator()(const Point2f& p)
+    {
+        return isnan(p.x) || isnan(p.y);
```
Comment (alalek) [inline modules/tracking/src/trackerMedianFlow.cpp]: What is about `cvIsNaN()`?

**Comment by LeonidBeynenson** [comment]
:+1:

**Pull request merged**